### ISSUE TITLE
status: show progress when doing byte-level checks

### ISSUE THREAD

**Issue by timmattison** [kevin-hanselman/dud#198]
This is related to #115

Since the full status check can take a long time it would be good to have a status bar that shows how far along it is and what is being done. I have a system that has a base level of data that is 275 GB. The data was updated (went from symlinks to regular files) but the contents were still the same. When I run `dud status` I can see that it has read 1.06 TB from disk and isn't finished yet so it'd be good to see what's going on so I can figure out if I did something wrong and/or if I want to wait for the results or do something else.

**Comment by kevin-hanselman** [comment]
I agree, this is a good feature! I'll look into how easy it would be to implement.

**Comment by kevin-hanselman** [comment]
FYI @timmattison, I [started work on this a while back](https://github.com/kevin-hanselman/dud/tree/new_ui), but I hit a speed bump. I want to generally improve the progress UI, and [bubbletea](https://github.com/charmbracelet/bubbletea) is a natural choice. However, I noticed that some of the benchmarks suffered more than I'd like, and profiling showed that it was in large part from bubbletea's overhead. I'm hesitant to incur a performance hit.

I'm also focusing on another side project at the moment. For anyone interested in this work, feel free to check out the branch linked above and consider picking up where I left off. 😄
